tests/components/mqtt/test_init.py

```diff
@@ -2425,22 +2643,24 @@ async def test_debug_info_non_mqtt(
     )
     for device_class in DEVICE_CLASSES:
         entity_registry.async_get_or_create(
-            DOMAIN,
+            mqtt.DOMAIN,
```

Comment:
I assume this is a bugfix

Reply:
no it is not, dunno why I did not replace with domain here, tests pass. I'll have a look.